### PR TITLE
Fix time-millis warning/stacktrace

### PR DESCRIPTION
**Pull request by darongmean**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -32,7 +32,11 @@
   :profiles {:dev         {:dependencies
                            [[midje "1.9.1" :exclusions [org.clojure/clojure]]
                             [nubank/matcher-combinators "0.2.1"]
-                            [joda-time "2.10"]]}
+                            [joda-time "2.10"]
+                            [com.fzakaria/slf4j-timbre "0.3.12"] ; Forward SLF4J to Timbre
+                            [com.taoensso/timbre "4.10.0"]]
+                           :jvm-opts
+                           ["-DTIMBRE_LEVEL=:debug"]}
```
Comment (guillaume) [inline project.clj]: why do we have to pass `jvm-opts`?

**Comment by guillaume** [comment]
I think the fix looks good. We'll have to update the version of the library after this PR

Just read the avro spec and this time-millis represents a time of the day (regardless of the date) and it's worth noting that it isn't used in our schemas at the moment

Also, I wonder if it's worth forking this library further...

**Comment by youvere** [comment]
some improvements have been made about error handling, we could do a pr with those change in abracad

File: project.clj (hunk 1 above)
Comment (darongmean) [reply project.clj]: It is to set the log level to debug. The default log level is info, i think.

**Pull request merged**